static/resources/harness.js

```diff
@@ -402,6 +402,38 @@
 
     updateStatus('Posting results to server...');
 
+    try {
+      var body = JSON.stringify(results);
+
+      var client;
+      if ('XMLHttpRequest' in self) {
+        client = new XMLHttpRequest();
+      } else if ('ActiveXObject' in self) {
+        client = new ActiveXObject('Microsoft.XMLHTTP');
+      }
+
+      if (!client) {
+        updateStatus('Cannot upload results: XMLHttpRequest is not supported.');
+        return;
+      }
+
+      client.open('POST', '/api/results?for='+encodeURIComponent(location.href));
+      client.setRequestHeader('Content-Type', 'application/json;charset=UTF-8');
+      client.send(body);
+      client.onreadystatechange = function() {
+        if (client.readyState == 4) {
+          if (client.status >= 200 && client.status <= 299) {
+            updateStatus('Results uploaded. <a href="/results" id="submit">Submit to GitHub</a>');
```

Comment:
Shouldn't this link to the created PR?

Reply:
In this step, the results are only submitted to the server and haven't been uploaded to GitHub.  This is to allow the Selenium script to run the test without creating a monstrous amount of PRs and download the results afterwards.

Reply:
Oh right, I forgot how this all works :D